Tests/MacroTestingTests/MacroExamples/OptionSetMacro.swift

```diff
@@ -79,7 +79,8 @@ public struct OptionSetMacro {
   static func decodeExpansion(
     of attribute: AttributeSyntax,
     attachedTo decl: some DeclGroupSyntax,
-    in context: some MacroExpansionContext
+    in context: some MacroExpansionContext,
+    emitDiagnostics: Bool
```

Comment:
Added to de-duplicate the diagnostics emitted

Reply:
I've just take a quick look at this PR and isn't it this issue: https://github.com/apple/swift-syntax/pull/2337 ?

Reply:
Opened a PR to contribute this de-duplication directly to `swift-syntax`: https://github.com/apple/swift-syntax/pull/2353